sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,159 @@
+# Preamble
+
+**SIP Number:** 029
+
+**Title:** Bootstrapping sBTC Liquidity and Incentive Preservation
+
+**Authors:**
+- Alex Miller (alex@hiro.so)
+- Andre Serrano (andre@bitcoinl2labs.com)
+- Brittany Laughlin (brittany@lattice.vc)
+- Jesse Wiley (jesse@stacks.org)
+- Jude Nelson (jude@stacks.org)
+- Philip De Smedt (philip@stackingdao.com)
+- Tycho Onnasch (tycho@zestprotocol.com)
+- Will Corcoran (will@stacks.org)
+
+**Consideration:** Economics, Technical, Governance
+
+**Type:** Consensus (hard fork)
+
+**Status:** Draft
+
+**Created:** 2024-11-06
+
+**License:** BSD 2-Clause
+
+**Sign-off:**
+
+**Discussions-To:**
+- Stacks Forum Post: [Aligning with Bitcoin Halving and Incentives after Nakamoto](https://forum.stacks.org/t/aligning-with-bitcoin-halving-and-incentives-after-nakamoto/17668)
+
+
+# Abstract
+
+The first Stacks halving is expected to take place at Stacks block height 210,384, which is set to occur during Reward Cycle 100 in December 2024, cutting the STX block reward from 1,000 STX to 500 STX. This SIP proposes a modification to the emissions schedule given that the network is going through two major launches (Nakamoto and sBTC) which rely on predictable economic incentives. The proposed schedule modification and associated STX emission rate would create time for Nakamoto and sBTC to launch and settle in, but, being mindful of supply, would still result in an overall reduced target 2050 STX supply (0.19% lower) and a reduced tail emission rate (20% lower).
```

Comment:
> at Stacks block height 210,384
This is not correct. It is set for 210,384 Bitcoin blocks after the Stacks 2.0 starting height, 666,050, which is Bitcoin height 876,434.

Reply:
Noted. I will correct this now.